Velaptor/ReactableData/ViewPortSizeData.cs

```diff
@@ -0,0 +1,21 @@
+﻿// <copyright file="ViewPortSizeData.cs" company="KinsonDigital">
+// Copyright (c) KinsonDigital. All rights reserved.
+// </copyright>
+
+namespace Velaptor.ReactableData;
+
+/// <summary>
+/// Holds the size of the view port.
```

Comment:
Refactor - should be one word
```suggestion
/// Holds the size of the viewport.
```